src/main/java/scala_maven/ScalaMojoSupport.java

```diff
@@ -764,6 +784,10 @@ private String getToolClasspath() throws Exception {
             options.add(baseDir.getAbsolutePath());
             // we always ask for metrics because there is no easy way to know if it's a clean build or not
             options.add("-Ymetrics");
+            if (hydraVersionWithDashboard()) {
```

Comment:
Fair point. But since I needed it for testing already, and makes the plugin a bit more versatile (allows people to easily downgrade Hydra in case of an issue, without downgrading also the plugin) I'm inclined to leave it (the logic is fairly simple). WDYT?

Reply:
Alright, the logic is simple enough, so all good.